apps/src/applab/firebaseCounters.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+/* global Applab */
+
+const Firebase = require("firebase");
```

Comment:
Done. Can anything previously node-style `require`d be es6-style `import`ed instead, or does the Firebase npm author have to do something sophisticated to make this work? I thought I had seen this not work in some cases, but now I don't remember specifically.